relay-clients/client-polkadot/src/lib.rs

```diff
@@ -122,5 +122,5 @@ impl ChainWithTransactions for Polkadot {
 
 impl ChainWithRuntimeVersion for Polkadot {
 	const RUNTIME_VERSION: Option<SimpleRuntimeVersion> =
-		Some(SimpleRuntimeVersion { spec_version: 1_002_000, transaction_version: 25 });
+		Some(SimpleRuntimeVersion { spec_version: 1_003_003, transaction_version: 26 });
```

Comment:
I think this is not enough - this is `client-polkadot` and we dont run version guard for relaychains,
but Bulat reported BridgeHubPolkadot:
```
2024-10-15 11:20:11 +00 ERROR bridge: The BridgeHubPolkadot client is configured to use runtime version SimpleRuntimeVersion { spec_version: 1003000, transaction_version: 4 } and actual version is SimpleRuntimeVersion { spec_version: 1003003, transaction_version: 4 }. Aborting
```